src/analysis/lattice.h

```diff
@@ -102,6 +119,9 @@ class FiniteIntPowersetLattice {
 
   // Returns an instance of the bottom lattice element.
   Element getBottom();
+
+  // Returns a random lattice element. Used for fuzzing.
+  Element getRandom(Random& rand);
```

Comment:
Do we need this in the class? I'd hope the fuzzer can do this itself.

Reply:
Refactored fuzzer to do this instead. After some initial examination, I found that the interface of the current powerset lattices were sufficient to generate random lattice without accessing private memebers.